kmip/pie/objects.py

```diff
@@ -353,6 +372,7 @@ def __init__(self, algorithm, length, value, masks=None,
         self.value = value
```

Comment:
Add a TODO to verify format_type and value. I would like to see a check where if a user specifies X.509 as format type and value does not conform to that then an exception is raised.